nbs/01a_losses.ipynb

```diff
@@ -62,7 +62,15 @@
     "class BaseLoss():\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #8.            axis:int=-1, # The demension that has the dimension
</pre><p> </p><p>I don't understand this docment</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/muellerzr/fastai-docment-sprint/pull/87/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

Reply:
Pretty sure that's because it doesn't make sense.  And I didn't even spell it correctly lol thanks for all your great comments.  It is definitely helping me improve